plugin/pkg/admission/defaultingressclass/admission.go

```diff
@@ -21,13 +21,14 @@ import (
 	"fmt"
 	"io"
 
+	networkingv1 "k8s.io/api/networking/v1"
 	networkingv1beta1 "k8s.io/api/networking/v1beta1"
```

Comment:
what in this file is still using `networkingv1beta1 "k8s.io/api/networking/v1beta1"`?

Reply:
It is the IngressClass default annotations https://github.com/kubernetes/kubernetes/blob/962572b9dd819366b96c6373b0120166d26d36fb/plugin/pkg/admission/defaultingressclass/admission.go#L128-L131

Reply:
ack